sys/include/net/nanocoap.h

```diff
@@ -228,7 +233,8 @@ typedef struct {
  * For POST, PATCH and other non-idempotent methods, this is an additional
  * requirement introduced by the contract of this type.
  */
-typedef ssize_t (*coap_handler_t)(coap_pkt_t *pkt, uint8_t *buf, size_t len, void *context);
+typedef ssize_t (*coap_handler_t)(coap_pkt_t *pkt, uint8_t *buf, size_t len,
+                                  coap_resource_ctx_t *context);
```

Comment:
Do you not like `coap_request_ctx_t` :) ? I don´t see how it would make sense to retrieve endpoint addresses from a resource context.

Reply:
Ah yes that makes more sense, also when naming the accessor functions